src/test/java/core/basesyntax/strategy/BalanceFruitHandlerTest.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax.strategy;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import org.junit.Before;
+import org.junit.Test;
+
+public class BalanceFruitHandlerTest {
+    private static final String BANANA = "banana";
+    private static final String OPERATION_BANANA = "b";
+    private static final Integer VALID_VALUE = 10;
+    private static final FruitTransaction TRANSACTION = new FruitTransaction(null, null, -1);
+    private static FruitHandler fruitHandler;
+
+    @Before
+    public void setUp() {
+        fruitHandler = new BalanceFruitHandler();
```

Comment:
move init of handler to BeforeClass in all 4 handlerTests
there is no need to do it before each test